benchmarks/twitter-finagle/src/main/scala/org/renaissance/twitter/finagle/FinagleChirper.scala

```diff
@@ -225,12 +226,11 @@ class FinagleChirper extends RenaissanceBenchmark {
   class Cache(val index: Int, val service: Service[Request, Response])
     extends Service[Request, Response] {
     val lock = new AnyRef
-    val cache = new mutable.HashMap[String, Buf]
-    var count = 0
+    val cache = new concurrent.TrieMap[String, Buf]
```

Comment:
Technically speaking, there is a race condition in accessing the `cache`, but that's probably not very important for the benchmark overall.